packages/http-client-python/eng/scripts/Generate-WithPyodide.ps1

```diff
@@ -0,0 +1,11 @@
+#Requires -Version 7.0
+
+Import-Module "$PSScriptRoot\Generation.psm1" -DisableNameChecking -Force;
+
+$repoRoot = Resolve-Path (Join-Path $PSScriptRoot '..' '..')
+
+Write-Host "Building project ..."
+& npm run build
+
+Write-Host "Regenerating project with Pyodide ..."
```

Comment:
I think there is no need to add new step in eng to run test twice with python and pyodide mode. We can run azure test with python mode and unbranded test with pyodide mode. Then you just need to add `use-pyodide: true ` here https://github.com/microsoft/typespec/blob/53a475d99db156271544871dc2ee881744d4f02a/packages/http-client-python/eng/scripts/ci/regenerate.ts#L284.

Reply:
we just want to make sure that generating with pyodide and with python result in the same output, so I think it is necessary to run twice